### PR TITLE
[IMP] inventory: update integrating_landed_costs

### PR DESCRIPTION
**Pull request by brse-odoo**
This PR is a **targeted revision** for **V14.0 only**.

Performed complete rewrite for Inventory document: integrating_landed_costs.rst

-Added new sections to document
-Fixed and replaced all screenshots
-Removed use of 2nd person
-Reworked example flow in doc

Cleaned up RST, including:
-Fixed 100th character line breaks
-Fixed indentations
-Fixed typos and improved grammar

### CODE REVIEW

**Comment by brse-odoo** [comment]
@StraubCreative, this is ready for your review whenever you have a chance. Thank you!

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @brse-odoo 👋 Ive made some technical changes on this doc, stated below for your review. I will be pushing these changes in a second commit for your approval, once approved I will squash the commits, rebase, and push them back up for final review.

This doc is written very well, my only changes were on 2 things:

1. :guilabel: use on general terms like "landed costs", "vender bills", and "inventory valuation"
2. Indenting within admonition blocks.

Please let me know if you approve of these changes and Ill forward this onto final review. Thank you 🙂

**Comment by brse-odoo** [comment]
> Hi @brse-odoo 👋 Ive made some technical changes on this doc, stated below for your review. I will be pushing these changes in a second commit for your approval, once approved I will squash the commits, rebase, and push them back up for final review.
> 
> This doc is written very well, my only changes were on 2 things:
> 
> 1. :guilabel: use on general terms like "landed costs", "vender bills", and "inventory valuation"
> 2. Indenting within admonition blocks.
> 
> Please let me know if you approve of these changes and Ill forward this onto final review. Thank you 🙂

Hi @samueljlieber, thanks for completing the technical review and implementing these changes! Everything looks good to me and you have my approval on the changes.

**Comment by samueljlieber** [comment]
Hi @brse-odoo, thanks for approving 🙂 I squashed the changes and rebased in https://github.com/odoo/documentation/commit/a825b6a33664dc54b4d42b83480458290abcfc65.

All set for your final review @StraubCreative 😎

**Review by samueljlieber — APPROVED** [review]

**Comment by brse-odoo** [comment]
Hi @odoo/us-doc-review, this PR could use a content review whenever one of you has a chance. Thank you!

**Sending through a content review check before sending to zst for final review. Will squash commits/rebase/push up changes and re-request final review when ready.**

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst [hunk 1]
```diff
@@ -1,64 +1,112 @@
...
-.. image:: integrating_landed_costs/integrating_landed_costs_05.png
-    :align: center
+Once all the information on the bill has been filled out, click :guilabel:`Create Landed Costs`.
+This will navigate to a new accounting entry for the *Landed Cost*. From here, next to
+:guilabel:`Transfers`, the warehouse receipt with which this landed cost is associated must be
+specified. Additionally, an expenses :guilabel:`Account` must be selected, as well.
+
+To see how the landed costs affect the items on the purchase order, click the :guilabel:`Valuation
+Adjustments` tab. From here, the :guilabel:`Original Value` and :guilabel:`New Value` can be seen,
+as well as the :guilabel:`Additional Landed Cost` that changed the original purchase order.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:92]: I can see these columns (Original Value and New Value), but everything under the Valuation Adjustments tab is blank. I've made sure to use a FIFO product, etc., but perhaps I did something incorrect along the way? It's worth looking into - and I'm happy to give this workflow another try if something needs to be corrected/adjusted.

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst [hunk 2]
```diff
@@ -1,64 +1,112 @@
...
+
+.. image:: integrating_landed_costs/integrating-landed-costs-valuation-adjustments.png
+   :align: center
+   :alt: Valuation adjustments tab for landed costs on vendor bill.
+
+Once everything is set, :guilabel:`Validate` and :guilabel:`Save`. Then, click back to the
+vendor bill (via the breadcrumbs).
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:99]: When I attempted to click 'Validate' I got the following error message: You cannot apply landed costs on the chosen Transfers(s). Landed costs can only be applied for products with FIFO or average costing method.

However, I made sure that FIFO was applied to the Product Category that was applied to BOTH: the product initially added to the RfQ & to the Service Type 'Landed Cost' product added during the Vendor Bill Draft page section. So, perhaps this is a bug? Did I misunderstand a step along the way? Worth looking into either way.

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst [hunk 3]
```diff
@@ -1,64 +1,112 @@
...
 
 .. note::
-   You are not forced to start from the vendor bill, you can also go in :menuselection:`Inventory
-   --> Operations --> Landed Costs` and directly create the landed cost from there.
+   When the :guilabel:`Default Split Method` is set to :guilabel:`Equal`, the landed cost will be
+   split equally between all items included on the purchase order. Since this :abbr:`PO (Purchase
+   Order)` has just one item, the full amount will be applied to this item.
+
+To wrap up the vendor bill, :guilabel:`Confirm`, :guilabel:`Register Payment`, and :guilabel:`Create
+Payment`.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:107]: Was unable to reach this portion, as the above steps caused an error pop-up.

**Review by ksc-odoo — COMMENTED** [review]
Hey @brse-odoo just finished a peer review of this doc, but I ran into some issues near the end of the flow. If you could give those steps a quick "double-check" that would be great. If I made an error along the way, or if a change was made, just let me know and I'll happily give this another try. Just let me know ... Other than that - great work!

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst [hunk 4]
```diff
@@ -1,64 +1,155 @@
...
-tick the box *Landed Costs* on the vendor bill line.
+To create a landed cost product, first navigate to :menuselection:`Inventory app --> Products -->
+Products`, and click the :guilabel:`Create` button, which reveals a blank product template page.
 
-.. image:: integrating_landed_costs/integrating_landed_costs_02.png
-    :align: center
+On the blank product template form, fill out the necessary product information, and change the
+:guilabel:`Product Type` to :guilabel:`Service`.
+
+Then, click the :guilabel:`Purchase` tab, and under the :guilabel:`Landed Costs` section, select
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:58]: 100th character

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst [hunk 5]
```diff
@@ -1,64 +1,155 @@
...
+the checkbox beside :guilabel:`Is a Landed Cost`. Once selected, a new field appears, in which a
+:guilabel:`Default Split Method` can be selected.
+
+Clicking the blank field beside :guilabel:`Default Split Method` reveals a drop-down menu. From
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:62]: 100th character

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst [hunk 6]
```diff
@@ -1,64 +1,155 @@
...
-If this product is always a landed cost, you can also define it on the
-product and avoid having to tick the box on each vendor bill.
+.. image:: integrating_landed_costs/integrating-landed-costs-split-method.png
+   :align: center
+   :alt: Landed costs and split method enabled on product form.
 
-.. image:: integrating_landed_costs/integrating_landed_costs_03.png
-    :align: center
+.. tip::
+   Depending on the landed cost of each shipment, the price of a landed cost product can change
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:75]: 100th character

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst [hunk 7]
```diff
@@ -1,64 +1,155 @@
...
-by clicking on the journal entry.
+Then, fill out the remaining information on the new :abbr:`RFQ (request for quotation)`. Lastly,
+click :guilabel:`Confirm Order` to confirm the order. Once the quotation has been confirmed, it
+turns into a :abbr:`PO (purchase order)`.
 
-.. image:: integrating_landed_costs/integrating_landed_costs_05.png
-    :align: center
+Next, click :guilabel:`Receive Products` to receive the ordered products, then click
+:guilabel:`Validate`, which reveals a pop-up. In the pop-up, select :guilabel:`Apply` to set the
+:guilabel:`Done` quantities. Then, navigate back to the initial :abbr:`PO (purchase order)` (via
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:95]: 100th character

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst [hunk 8]
```diff
@@ -1,64 +1,155 @@
...
+
+Once goods have been received from the purchase order, a vendor bill can be created directly from
+the :abbr:`PO (purchase order)`. To do so, click :guilabel:`Create Bill`. Doing so reveals a vendor
+bill draft page.
+
+On this vendor bill draft page, the information from the :abbr:`PO (purchase order)` is
+automatically populated. To add a landed cost to the bill, first click :guilabel:`Edit`.
+
+Next, click :guilabel:`Add a line` in the :guilabel:`Invoice Lines` tab to add the newly-created
+landed cost product to the vendor bill. And, since this product was specifically created as a
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:109]: 100th character

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst [hunk 9]
```diff
@@ -1,64 +1,155 @@
...
+   :alt: Landed cost added on product lines on vendor bill.
+
+.. important::
+   Since no :guilabel:`Price` was set on the product template form for the landed cost product, it
+   **must** be set manually on the vendor bill, in the :guilabel:`Price` column.
+
+Create landed costs
+===================
+
+Once all the necessary information on the bill has been filled out, click
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:123]: 100th character

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst [hunk 10]
```diff
@@ -1,64 +1,155 @@
...
+:guilabel:`Create Landed Costs` at the top of the bill.
+
+Doing so reveals a separate page, wherein the necessary landed cost information is entered. From
+this page, next to the greyed-out :guilabel:`Transfers` field, the warehouse receipt that this
+landed cost is associated with must be specified. To adjust the :guilabel:`Transfers` field, click
+:guilabel:`Edit` to make that field accessible.
+
+Additionally, an expenses :guilabel:`Account` (located in the :guilabel:`Additional Costs` tab)
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:131]: 100th character

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst [hunk 11]
```diff
@@ -1,64 +1,155 @@
...
+must be selected, as well.
+
+To see how the landed costs affect the items on the :abbr:`PO (purchase order)`, click the
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:134]: 100th character

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst [hunk 12]
```diff
@@ -1,64 +1,155 @@
...
+:guilabel:`Valuation Adjustments` tab. From this tab, the :guilabel:`Original Value` of the
+purchase order and the :guilabel:`New Value` of the order can be seen and compared, as well as the
+:guilabel:`Additional Landed Cost` that was added to change the original
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:137]: 100th character

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @brse-odoo 
Please fix all of the 100th character line break issues and tag me again for another review.
These should have not made it past technical review 😉 
Thank you.

cc: @samueljlieber

**Comment by brse-odoo** [comment]
Removed file `product_variants` in latest commit `4731e2e` to resolve merge conflicts/ci check failures. Pushed up changes and will review doc once more before sending to review.

**Comment by brse-odoo** [comment]
Hi @StraubCreative this PR is ready for final review when you have the chance, thank you 👍

**Review by StraubCreative — APPROVED** [review]
Hi @brse-odoo 
Good PR. 
Approving with comments for you to address.
When you're done, you can run the merge command.
Thanks!

@fw-bot ignore
@robodoo delegate=brse-odoo

**Pull request closed without merging**